src/rattler_build_conda_compat/jinja/objects.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+
+class _StubEnv:
+    """A class to represent the env object used in rattler-build recipe."""
+
+    def get(self, env_var: str, default: str | None = None) -> str:
+        if default:
+            return f"""${{{{ env.get("{env_var}", default="{default}") }}}}"""
+
+        return f"""${{{{ env.get("{env_var}")}}}}"""
```

Comment:
If I understand correctly, this just returns the jinja string _as is_? Fine with me.

Reply:
made them all to return just the simple string